tidb-cloud/migrate-from-oracle-using-aws-dms.md

```diff
@@ -0,0 +1,135 @@
+
+## Step 4. Create TiDB Cloud Serverless Tier
+
+After registering and logging in, you can create a free serverless TiDB tier.
+
+![Create TiDB Cloud Serverless Tier](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-4.png)
+
+After the cluster is created, you will see the following, and then click **Security Settings** to set the user password.
+
+![Set TiDB Cloud Serverless Tier Password](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-5.png)
+
+![Set TiDB Cloud Serverless Tier Password](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-6.png)
+
+After you fill the password, click the generate and copy to generate password, and importantly remember the generated password. Then, click the submit button.
+
+Next, click the Connect button, you can see how to connect the serverless TiDB:
+
+![Connect to TiDB Cloud Serverless Tier](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-7.png)
+
+## Step 5. Create AWS DMS Replication Instance
+
+Create AWS DMS Replication instance with `dms.t3.large` in VPC.
+
+![Create AWS DMS Instance](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-8.png)
+
+## Step 6. Create DMS Endpoint
+
+Create Oracle source endpoint and TiDB target endpoint.
+
+![Create AWS DMS Source endpoint](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-9.png)
+
+![Create AWS DMS Source endpoint](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-10.png)
+
+## Step 7. Migrating Schema
+
+As the [DMS official documentation](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_GettingStarted.SCT.html), we may need to migrate schema using [AWS Schema Conversion Tool](https://docs.aws.amazon.com/SchemaConversionTool/latest/userguide/CHAP_Installing.html#CHAP_Installing.Procedure), but in this example we will let DMS auto handle the schema, since the schema definition is pretty simple.
+
+## Step 8. Create DMS Database Migration Task
+
+On AWS Management Console, Create DMS Migration Task and specify the **Selection rules**:
+
+![Create AWS DMS migration task](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-11.png)
+
+![AWS DMS migration task selection rules](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-12.png)
+
+Then create the task, start it, and then wait for the task to finish.
+
+Then click the **Table statistics** to see the synced table, remember the schema name is ADMIN:
+
+![Check AWS DMS migration task](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-13.png)
+
+## Step 9. Check data In Downstream TiDB Cloud Serverless Tier
+
+Connect to [TiDB Cloud Serverless Tier](https://tidbcloud.com/console/clusters/create-cluster) and check the `admin.github_event` table data, as you can see, DMS successfully synced table github_events and 10000 rows data.
+
+![Check Data In TiDB](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-14.png)
+
+## Conclusion
+
+With AWS DMS, you can successfully migrate data from upstream AWS RDS(MySQL, Oracle) to [TiDB Cloud Serverless Tier](https://tidbcloud.com/console/clusters/create-cluster) With the following exceptions:
```

Comment:
Probably change  - you can successfully migrate data from upstream AWS RDS(MySQL, Oracle) 
**to**
you can successfully migrate data from ANY upstream AWS RDS database in this case we used Oracle

Reply:
done